server/mlcube/models.py

```diff
@@ -10,7 +10,9 @@ class MlCube(models.Model):
 
     name = models.CharField(max_length=20, unique=True)
     git_mlcube_url = models.CharField(max_length=256)
-    git_parameters_url = models.CharField(max_length=256)
+    mlcube_hash = models.CharField(max_length=100)
```

Comment:
I see two migrations: Is it because you tried changing the field attributes in two steps?

Reply:
True. I will merge them